src/app/lib/views/browser/list.js

```diff
@@ -370,61 +346,22 @@
             }
         },
 
-        moveUp: function (e) {
-            e.preventDefault();
-            e.stopPropagation();
-
-            var index = $('.items .item.selected').index();
-            if (index === -1) {
-                index = 0;
-            } else {
-                index = index - this.itemsPerRow();
-            }
-            if (index < 0) {
-                return;
-            }
-            this.selectIndex(index);
-        },
-
-        moveDown: function (e) {
-            e.preventDefault();
-            e.stopPropagation();
-
-            var index = $('.items .item.selected').index();
-            if (index === -1) {
-                index = 0;
-            } else {
-                index = index + this.itemsPerRow();
-            }
-            this.selectIndex(index);
-        },
-
-        moveLeft: function (e) {
+        move: function (e) {
             e.preventDefault();
             e.stopPropagation();
 
+            var row = this.itemsPerRow();
             var index = $('.items .item.selected').index();
-            if (index === -1) {
-                index = 0;
-            } else if (index === 0) {
-                index = 0;
-            } else {
-                index = index - 1;
-            }
-            this.selectIndex(index);
-        },
+            index < 0 && (index = 0);
```

Comment:
i'd rather have a check for `index === -1` as what we're checking is if we had a match